packages/cos-setup/definition.yaml

```diff
@@ -1,6 +1,6 @@
 name: cos-setup
 category: system
-version: "0.6"
+version: "0.6.1"
```

Comment:
requires reverse bump?

Reply:
mmm reverse bump... 😓 Probably because it's Monday morning, but I don't get what are you referring to... I just wanted to make a version bump, I though this was a patch level change but when I wanted to update the version I noticed this package not follow the common semver schema of MAJ.MIN.PTCH, so I added the patch level number.

Reply:
I meant bumping the cos packages so they include this one, otherwise all the cos-system, cos-img, etc... packages are using the old cos-setup version unless we bump those no?

Reply:
Oh, I see, I think you are right.